src/lxc/commands_utils.c

```diff
@@ -117,7 +117,7 @@ int lxc_make_abstract_socket_name(char *path, size_t pathlen,
 	}
 
 	ret = snprintf(offset, len, "%s/%s/%s", lxcpath, name, suffix);
-	if (ret < 0)
+	if (ret < 0 || (size_t)ret >= len)
```

Comment:
So this is intentional. If you look below you'll see that overflow is handled further below by falling back to hashing.

Reply:
thanks...